monkey/infection_monkey/exploit/zerologon.py

```diff
@@ -189,9 +190,8 @@ def assess_exploit_attempt_result(self, exploit_attempt_result) -> bool:
                 logger.info(error_message)
 
                 self._publish_exploitation_event(
-                    target=self.host.ip_addr,
-                    exploitation_success=False,
-                    tags=EXPLOIT_TAGS,
+                    time=time(),
```

Comment:
I think you don't need `time=time()` as `time()` is already a default when publishing an events.